portlets/opencps-portlet/docroot/WEB-INF/src/org/opencps/usermgt/portlet/UserMgtPortlet.java

```diff
@@ -1063,36 +872,30 @@ else if (workingUnit != null && workingUnitId > 0 && workingUnit
 		try {
 
 			parentWorkingUnit = WorkingUnitLocalServiceUtil
-			    .fetchWorkingUnit(parentWorkingUnitId);
+					.fetchWorkingUnit(parentWorkingUnitId);
 
-		}
-		catch (Exception e) {
-			// nothing to do
+		} catch (Exception e) {
+			_log.error(e);
 		}
 
-		if (parentWorkingUnit != null && !parentWorkingUnit
-		    .getIsEmployer() && isEmployer) {
+		if (parentWorkingUnit != null && !parentWorkingUnit.getIsEmployer()
+				&& isEmployer) {
 			throw new OutOfScopeException();
-		}
-		else if (Validator
-		    .isNotNull(govAgencyCode) && !isEmployer) {
+		} else if (Validator.isNotNull(govAgencyCode) && !isEmployer) {
 			throw new OutOfScopeException();
 		}
 
 		WorkingUnit getWorkingUnitByEmail = null;
 		try {
 			getWorkingUnitByEmail = WorkingUnitLocalServiceUtil
-			    .getWorkingUnitByEmail(email);
-		}
-		catch (Exception e) {
-
+					.getWorkingUnitByEmail(email);
+		} catch (Exception e) {
+			_log.error(e);
 		}
 		if (getWorkingUnitByEmail != null && workingUnitId <= 0) {
 			throw new DuplicatWorkingUnitEmailException();
-		}
-		else if (getWorkingUnitByEmail != null && workingUnitId > 0 &&
-		    getWorkingUnitByEmail
-		        .getWorkingunitId() != workingUnitId) {
+		} else if (getWorkingUnitByEmail != null && workingUnitId > 0
+				&& getWorkingUnitByEmail.getWorkingunitId() != workingUnitId) {
 			throw new DuplicatWorkingUnitEmailException();
 		}
 	}
```

Comment:
@huymq review lại hộ anh file này phát, yêu cầu trình bày logic các hàm, toàn bộ phần quản trị cán bộ nằm trong đây.

Phần này sửa rồi hay lặp lại lỗi.